notification-config-service-impl/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    `java-library`
+    jacoco
+    id("org.hypertrace.jacoco-report-plugin")
+}
+
+dependencies {
+    api(projects.notificationConfigServiceApi)
+    implementation(projects.configServiceApi)
+    implementation(projects.configProtoConverter)
+    implementation(projects.objectStore)
+    implementation(projects.validationUtils)
+
+    implementation(libs.guava)
+    implementation(libs.rxjava3)
```

Comment:
make sure we're only declaring libraries being used, don't see rx being used.

Reply:
+1